src/search1_conversion/convert_result.py

```diff
@@ -167,26 +167,20 @@ def _fetch_narrative_info(results, meta):
     search_params['query'] = {
         'bool': {'should': matches}
     }
-    # Make the query for narratives on ES
-    search_results = es_client.search(search_params, meta)
```

Comment:
I'm trying to follow along with what changed here.  So instead of doing a separate query, you are just getting this info from the ws object record?  It seems fine, but I'm trying to understand why it used elastic before if it already had the info.